paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/AfterpayClearpayDefinitionTest.kt

```diff
@@ -0,0 +1,50 @@
+package com.stripe.android.lpmfoundations.paymentmethod
+
+import androidx.test.core.app.ApplicationProvider
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.model.PaymentIntentFixtures
+import com.stripe.android.testing.LocaleTestRule
+import com.stripe.android.ui.core.R
+import org.junit.Rule
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.util.Locale
+import kotlin.test.Test
+
+@RunWith(RobolectricTestRunner::class)
+internal class AfterpayClearpayDefinitionTest {
+    @get:Rule
+    val localeRule = LocaleTestRule()
+
+    @Test
+    fun `Test label for afterpay show correctly when clearpay string`() {
+        localeRule.setTemporarily(Locale.UK)
+        val paymentMethodMetadata = PaymentMethodMetadataFactory.create(
```

Comment:
It'd be nice to extract out the stuff that's shared between these two tests so that it's really obvious what differs. E.g. it looks like getting the supportedPaymentMethod display name is the exact same in these tests, so you could extract that to a helper. Then it's really obvious that the only difference between the tests is:
- the locale
- the final display name

Reply:
Good suggestion, updated.